test_prebuild/test_unit_conv/unit_conv_scheme_1.F90

```diff
@@ -0,0 +1,55 @@
+!>\file unit_conv_scheme_1.F90
+!! This file contains a unit_conv_scheme_1 CCPP scheme that does nothing
+!! except requesting the minimum, mandatory variables.
+
+module unit_conv_scheme_1
+
+   use, intrinsic :: iso_fortran_env, only: error_unit
+   use ccpp_kinds, only : kind_phys
+   implicit none
+
+   private
+   public :: unit_conv_scheme_1_run
+
+   !! This is for unit testing only
+   real(kind_phys), parameter :: target_value = 1.0_kind_phys
+
+   contains
+
+!! \section arg_table_unit_conv_scheme_1_run Argument Table
+!! \htmlinclude unit_conv_scheme_1_run.html
+!!
+   subroutine unit_conv_scheme_1_run(data_array, data_array_opt, errmsg, errflg)
+      character(len=*), intent(out)   :: errmsg
+      integer,          intent(out)   :: errflg
+      real(kind_phys),  intent(inout) :: data_array(:)
+      real(kind_phys),  intent(inout), optional :: data_array_opt(:)
+
+      ! Initialize CCPP error handling variables
+      errmsg = ''
+      errflg = 0
+      ! Check values in data array
+      write(error_unit,'(a,e12.4))') 'In unit_conv_scheme_1_run: checking min/max values of data array to be approximately ', target_value
```

Comment:
It looks like there's a mismatched parenthesis here? It produces a warning with ifort but I am surprised it's not a syntax error.